ui/yarn.lock

```diff
@@ -5037,6 +5059,14 @@ insert-css@^2.0.0:
   resolved "https://registry.yarnpkg.com/insert-css/-/insert-css-2.0.0.tgz#eb5d1097b7542f4c79ea3060d3aee07d053880f4"
   integrity sha1-610Ql7dUL0x56jBg067gfQU4gPQ=
 
+inspire-schemas@^61.2.6-2-g90a8aaf:
+  version "61.2.6-2-g90a8aaf"
+  resolved "https://registry.yarnpkg.com/inspire-schemas/-/inspire-schemas-61.2.6-2-g90a8aaf.tgz#bb12cf7b20a0aabdb42d443f6aaed32de0ebae7b"
+  integrity sha512-tt3ZeV84xK+F+G7uoA4IMWN6HWTx+12uxoWhpgMlds6GYQpFziEgXBfccr6wXX9iYCdEKvQ8jvNIxCLIr5oK7g==
+  dependencies:
```

Comment:
that seems wrong, `inspire-schemas` shouldn't have any dependencies, I think all those should have been a `devDependency`